libraries/apollo-gradle-plugin-external/src/main/kotlin/com/apollographql/apollo3/gradle/api/Service.kt

```diff
@@ -426,6 +426,22 @@ interface Service {
    */
   val generateOptionalOperationVariables: Property<Boolean>
 
+  /**
+   * Specifies which methods will be auto generated on operations, models, fragments and input objects.
+   *
+   * Pass a list of any of the following:
+   *
+   * - "equalsHashCode" generates `equals` and `hashCode` methods that will compare generated class properties
```

Comment:
Note that `componentN` is not included as an option here. I think we can add it later if somebody asks for it.